pkg/plugins/policies/meshhttproute/xds/filters/requestheadermodifier.go

```diff
@@ -0,0 +1,70 @@
+package filters
+
+import (
+	"strings"
+
+	envoy_config_core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoy_route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+
+	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
+	api "github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute/api/v1alpha1"
+	util_proto "github.com/kumahq/kuma/pkg/util/proto"
+)
+
+type RequestHeaderModifierConfigurer struct {
+	headerModifier *api.HeaderModifier
```

Comment:
all of the configurers can just take non-pointers

Reply:
Fixed, since I added `switch` statement, there is no need in these pointers